azerothcore/azerothcore-wotlk
# fix(Core/Dungeon): Halls of stone doors

## CHANGES PROPOSED:
-  Closes the door before both starting bosses die

## ISSUES ADDRESSED:
- Closes #59 



## TESTS PERFORMED:
- Built and tested in-game




## HOW TO TEST THE CHANGES:

- .go creature 126743
- Check if door is closed
- .go creature 126789
- .die
- go creature 126790
- .die
- .go creature 126743
- Check if door is open

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
